rust/osm2lanes/src/transform/tags_to_lanes/bus.rs

```diff
@@ -37,13 +35,9 @@ pub(super) fn bus(
             .is_some(),
     ) {
         (false, false, false) => {}
-        (true, _, false) => busway(tags, locale, oneway, forward_side, backward_side, warnings)?,
-        (false, true, false) => {
-            lanes_bus(tags, locale, oneway, forward_side, backward_side, warnings)?
-        }
-        (false, false, true) => {
-            bus_lanes(tags, locale, oneway, forward_side, backward_side, warnings)?
-        }
+        (true, _, false) => busway(tags, locale, road, warnings)?,
```

Comment:
It's funny how just shrinking things to one line makes it easier to understand. The importance of visual structure to quickly scanning code...